avm/ptn/ai-platform/baseline/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -11,8 +11,9 @@ metadata description = 'This instance deploys the module in alignment with the b
 @maxLength(90)
 param resourceGroupName string = 'dep-${namePrefix}-aiplatform-baseline-${serviceShort}-rg'
 
-@description('Optional. The location to deploy resources to.')
-param resourceLocation string = deployment().location
+// Enforce uksouth to avoid restrictions around VM zones in certain regions
+#disable-next-line no-hardcoded-location
+var enforcedLocation = 'uksouth'
```

Comment:
nit: Wondering whether it's worth picking one of the major regions if we are using it for tests. Coming at this from a capacity angle - e.g. does westeurope do what you need?

Reply:
By default it deploys to westeurope but that's where there are the capacity limitations

uksouth is one of the recommended regions so I think it's fine

https://github.com/Azure/bicep-registry-modules/blob/main/avm/utilities/pipelines/e2eValidation/regionSelector/Get-AvailableResourceLocation.ps1#L36-L41